src/server/pfs/server/server_test.go

```diff
@@ -2310,17 +2308,17 @@ const (
 var (
 	port int32 = 30653 // Initial port on which pachd server processes will serve
 
-	startPFSServersOnce sync.Once // ensure pachd servers are only started once
+	checkEtcdOnce sync.Once // ensure we only test the etcd connection once
 )
 
-// src/server/pfs/server/driver.go expects an etcd server at "localhost:32379"
-// Try to establish a connection before proceeding with the test (which will
-// fail if the connection can't be established)
-func startPFSServers(t *testing.T) {
-	startPFSServersOnce.Do(func() {
+func getClient(t *testing.T) *pclient.APIClient {
+	// src/server/pfs/server/driver.go expects an etcd server at "localhost:32379"
```

Comment:
I believe etcd is actually at on 2379 and 32379 is the port we expose with node ports.
I could be wrong about that though, or it could be that this comment is just wrong, or it actually is using 32379 when it should be 2379 in which case presumably CI would catch it. Do you happen to know which of those it is?

Reply:
IIUC, the comment is wrong. Basically:
- this test runs outside kubernetes (typically), but uses etcd (as it runs a PFS server in a separate process) so it has to talk to etcd on 32379.
- We had a `NewLocalDriver` function in `driver.go` that basically existed to satisfy this test. Therefore `NewLocalDriver` which is technically in `driver.go`, expected an etcd server at `32379`
- However, `NewDriver`, which is what regular `pachd` uses, expects a server at `2379`.
- Since `NewLocalDriver` was deleted in this PR, there's nothing in `driver.go` that expects an etcd server at `32379` anymore, so this comment should be deleted.